docs/server/source/production-deployment-template/add-node-on-kubernetes.rst

```diff
@@ -163,16 +286,27 @@ You can SSH to an existing BigchainDB instance and run the ``bigchaindb
 show-config`` command to check that the keyring is updated.
 
 
-Step 7: Run NGINX as a Deployment
----------------------------------
-
-Please see :ref:`this page <Step 9: Start the NGINX Kubernetes Deployment>` to
-set up NGINX in your new node.
-
-
-Step 8: Test Your New BigchainDB Node
+Step 9: Test Your New BigchainDB Node
 -------------------------------------
 
 Please refer to the testing steps :ref:`here <Step 19: Verify the BigchainDB
 Node Setup>` to verify that your new BigchainDB node is working as expected.
 
+
+Step 10: Monitoring and Backup agent(Optional)
+----------------------------------------------
+
+This section is optional, but *if* you want to deploy MongoDB monitoring and backup agent for the new
+cluster, you have to authenticate each agent individually using its unique certificate. For more
+information about how to authenticate and add users to MongoDB please refer to:
+
+* :ref:`Configure Users and Access Control for MongoDB<Step 13: Configure Users and Access Control for MongoDB>`
+
+
+Step 11: Start OpenResty Service and Deployment(Optional)
+---------------------------------------------------------
```

Comment:
I understand that this will be centralised system if we have one OpenResty instance but I kept it optional so that a user doesn't think things won't be operational if we do not do this. Keeping multiple instance is ofcourse a better deployment configuration. I will edit this. Same for monitoring and backup agent.